completions/nvm.fish

```diff
@@ -3,15 +3,15 @@ set commands help install uninstall use run current ls ls-remote version \
 
 function __nvm_complete_ls_remote
   if not test "$__nvm_ls_remote"
-    set -g __nvm_ls_remote (fenv source ~/.nvm/nvm.sh\; nvm_ls_remote\; nvm_ls_remote_iojs)
+    set -g __nvm_ls_remote (fenv source $nvm_prefix/nvm.sh\; nvm_ls_remote\; nvm_ls_remote_iojs)
```

Comment:
what will happen if $nvm_prefix is not set?

Reply:
Looking in the code again, in [this](https://github.com/derekstavis/plugin-nvm/blob/master/init.fish) file, the `$nvm_prefix` variable is already been set.